mrmustard/lab_dev/wires.py

```diff
@@ -221,7 +246,8 @@ def __lshift__(self, other: Wires) -> Wires:
 
     @staticmethod
     def _outin(si, so, oi, oo):
-        r"""Returns the output and input wires of the composite object made by connecting
+        r"""
```

Comment:
@ziofil I'm a bit confused by this docstring. 
I would rephase it as:
""
Returns the ``ids`` of the composite object made by connecting an object ``self`` with input (output) ``ids`` ``si`` and ``so`` and another object ``other`` with ``ids`` ``oi`` and ``oo``. Assumes (but does not check) that ``self`` and ``other`` can indeed be connected
""

Does this make sense?

Reply:
Yes, I've taken your suggestion into account :+1: